dags/openshift_nightlies/dag.py

```diff
@@ -111,14 +112,35 @@ def build(self):
     def _get_openshift_installer(self):
         return jetski.BaremetalOpenshiftInstaller(self.dag, self.version, self.release_stream, self.platform, self.profile, self.openshift_build)
 
-        
+class ShiftOnStackNightlyDAG(AbstractOpenshiftNightlyDAG):
+    def __init__(self, version, release_stream, platform, profile, version_alias, build):
+        super().__init__(version, release_stream, platform, profile, version_alias)
+        self.release_stream_base_url = Variable.get("release_stream_base_url")
+        self.latest_release = var_loader.get_latest_release_from_stream(self.release_stream_base_url, self.release_stream)
```

Comment:
`self.release_stream_base_url` is set to https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/ so the release_stream passed through the manifest has to be one that exists from this URL. Moreover, `var_loader.get_latest_release_from_stream()` assumes a certain API spec from the `release_stream_base_url`. Setting the release stream to `latest-4.7` won't work with this. See https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/util/var_loader.py#L18

Reply:
removed the stream vars, will be using latest-4.7/4.8